src/applications/vaos/tests/new-appointment/components/VAFacilityPage/index.unit.spec.jsx

```diff
@@ -476,7 +477,8 @@ describe('VAOS <VAFacilityPage>', () => {
         .exist;
     });
 
-    it('should show additional info link if there are unsupported facilities within 100 miles', async () => {
+    // Temporarily disabling test to troubleshoot
+    it.skip('should show additional info link if there are unsupported facilities within 100 miles', async () => {
```

Comment:
Temporarily skip this test to isolate failures

Reply:
Since commenting out the section that fails, can remove the skip on this test